Content.Server/HotPotato/HotPotatoSystem.cs

```diff
@@ -0,0 +1,47 @@
+using Content.Shared.Hands.EntitySystems;
+using Content.Shared.Hands.Components;
+using Content.Server.Explosion.EntitySystems;
+using Content.Shared.Weapons.Melee.Events;
+using Content.Shared.HotPotato;
+
+namespace Content.Server.HotPotato;
+
+public sealed class HotPotatoSystem : SharedHotPotatoSystem
+{
+    [Dependency] private readonly SharedHandsSystem _hands = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<HotPotatoComponent, MeleeHitEvent>(TransferItem);
+        SubscribeLocalEvent<HotPotatoComponent, ActiveTimerTriggerEvent>(OnActiveTimer);
+    }
+
+    private void TransferItem(EntityUid uid, HotPotatoComponent comp, MeleeHitEvent args)
+    {
+        comp.CanTransfer = true;
+        TryTransferItem(uid, args);
+        comp.CanTransfer = !HasComp<ActiveHotPotatoComponent>(uid);
+    }
```

Comment:
Shared?

Reply:
if i put this to shared then item transfer animation will play twice